app/models/katello/glue/provider.rb

```diff
@@ -3,29 +3,11 @@ module Glue::Provider
     def self.included(base)
       base.send :include, InstanceMethods
       base.class_eval do
-        before_destroy :destroy_products_orchestration
+        before_destroy :del_products
       end
```

Comment:
Then we lose all of the [amazing logging and error handling that del_products provides](https://git.io/vXmor).